### PR TITLE
Added login message options

### PR DESCRIPTION
**Pull request by legoboy0215**

### CODE REVIEW

File: src/HereAuth/User/User.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
  *
  * Copyright (C) 2016 PEMapModder
  *
- * This program is free software: you can redistribute it and/or modify
+ * This program is free software: you can redistributeY it and/or modify
```
Comment (PEMapModder) [inline src/HereAuth/User/User.php]: what?

**Comment by legoboy0215** [comment]
Sorry XD

File: src/HereAuth/User/User.php [hunk 2]
```diff
@@ -177,7 +177,7 @@ public function onAuth(){ // TODO $method
 		$this->accountInfo->lastSkinHash = HereAuth::hash($this->getPlayer()->getSkinData(), $this->getPlayer()->getSkinName());
 		$this->accountInfo->lastIp = $this->getPlayer()->getAddress();
 		if($this->accountInfo->passwordHash){
-			$this->player->sendMessage("You have been authenticated.");
+			$this->player->sendMessage($this->getMain()->getMessages()->getNested("Login.Completion", "login"));
```
Comment (PEMapModder) [inline src/HereAuth/User/User.php:180]: There is no such message.

**Comment by legoboy0215** [comment]
There is?

File: src/HereAuth/User/User.php (hunk 2 above)
Comment (legoboy0215) [inline src/HereAuth/User/User.php:180]: Here you go ;)

**Comment by PEMapModder** [comment]
Originally I wanted to ask you to rebase the commits, but never mind.

**Comment by legoboy0215** [comment]
XD IDK how to use Git though.

**Pull request merged**